tests/unit_tests/EvalTests.cpp

```diff
@@ -15,4 +15,12 @@ TEST_CASE("Simple eval", "[eval") {
     CHECK(value.integer() == 13);
 }
 
+TEST_CASE("Function call", "[eval") {
```

Comment:
Yeah, but right now I only have support for return statements so there's not really anything to test there yet.